iron-a11y-keys-behavior.html

```diff
@@ -249,7 +249,9 @@
     Polymer.IronA11yKeysBehavior = {
       properties: {
         /**
-         * The HTMLElement that will be firing relevant KeyboardEvents.
+         * The HTMLElement that will be firing relevant KeyboardEvents. Set it to
```

Comment:
shouldn't this technically say "The *EventTarget* that will be ...".  is it valid to set this to `document` or `window` (which aren't `HTMLElement`s but are `EventTarget`s)?